frontend/packages/console-app/src/components/nodes/NodeLogs.tsx

```diff
@@ -0,0 +1,105 @@
+import * as React from 'react';
+import { Checkbox } from '@patternfly/react-core';
+import classnames from 'classnames';
+import { useTranslation } from 'react-i18next';
+import {
+  Dropdown,
+  getQueryArgument,
+  LoadingBox,
+  setQueryArgument,
+} from '@console/internal/components/utils';
+import { modelFor, NodeKind, resourceURL } from '@console/internal/module/k8s';
+
+type NodeLogsProps = {
+  obj: NodeKind;
+};
+
+const NodeLogs: React.FC<NodeLogsProps> = ({ obj: node }) => {
+  const dropdownItems = {
+    journal: 'journal',
```

Comment:
For `journal`, I think it will be important to be able to specify a unit. (See `oc adm node-logs -h`.)